### DISCUSSION TITLE
Is it a roadmap for EDC to integrate Walt.id.

### DISCUSSION THREAD

**Discussion by xdxxxdx** [eclipse-edc/.github#89] [Q&A] [answered]
Hello EDC team,
Just wondering if there is a plan for EDC to integrate to[ walt id](https://walt.id/) solution for participant management.
Thanks

**Comment by mspiekermann** [comment, marked as answer]
As a framework we are supporting multiple identity and wallet solutions by providing the corresponding interfaces e.g. for handling DID via various methods. Solution provider can adopt these to provide extensions that can be used by EDC implementations and we'd be happy to see them on our list of [known_friends](https://eclipse-edc.github.io/docs/#/documentation/KNOWN_FRIENDS). It is not planned to integrate walt.id out of the EDC committer team though.
